Greyci Gonzalez/problema13.c

```diff
@@ -0,0 +1,14 @@
+#include<stdio.h>
+void main(){

```

Comment:
El problema esta bien. Pero te falto usar el ciclo while... en las indicaciones se pedia hacerlo con for y con while